sympy/physics/mechanics/jointsmethod.py

```diff
@@ -0,0 +1,278 @@
+from sympy.physics.mechanics import (Body, Lagrangian, KanesMethod, LagrangesMethod,
```

Comment:
To mimic the kane and lagrange files, maybe this code should be moved into `joints.py` so that `from sympy.physics.joints import JointsMethod, PinJoint, PrismaticJoint` works cleanly.